lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -2410,6 +2410,9 @@ def post_create_table(self, table):
                 "\n TABLESPACE %s" % self.preparer.quote(tablespace_name)
             )
 
+        if pg_opts["using"]:
+            table_opts.append("\n USING %s" % pg_opts["using"])
```

Comment:
The order of these doesn't matter, right?

Reply:
Good catch. I just checked and it seems it's required to be at least before `WITH OIDS`. Updated: c165a547c7b2a87d63783f3154cd0bc3429f4b95.

Reply:
indeed is seems to be after partition by, at least according to the pg docs